specviz/widgets/workspace.py

```diff
@@ -357,6 +357,13 @@ def compose_filter_string(reader):
 
         return filters, loader_name_map
 
+    def display_load_data_error(self, exp):
+        message_box = QMessageBox()
```

Comment:
We may want to start using the `QMessageBox` class methods to generate dialogs, since they can be monkeypatched with qtbot for testing.

```python
    QMessageBox.critical(self, "Error loading data set.", str(exp))
```

Reply:
I can make that change here if you think it's appropriate.

Reply:
If it's not too much trouble, it'd save us in the future if the travis tests fail and we have run locally to figure out why (because travis won't tell you it's because it was hung up on an open dialog window).

Reply:
For some reason the title won't display when I set it up this way. 😕

Reply:
The title in the dialog box? (The "Error loading data set.")?

Reply:
We can skip making the change right now and explore it in the future.